core/src/allocation.jl

```diff
@@ -1128,7 +1113,9 @@ function allocate!(
         @debug JuMP.solution_summary(problem)
         if JuMP.termination_status(problem) !== JuMP.OPTIMAL
             (; allocation_network_id) = allocation_model
-            priority = priorities[priority_index]
+            priority = priorities[priority_idx]
+            println(JuMP.solution_summary(problem))
+            println(problem)
             error(
```

Comment:
We don't want these print statements on the main branch. The line starting with `@debug` above adds the solution summary to the logging data if Ribasim is run in debug mode. I don't know how to access the logging data, but it shouldn't be too hard.